schema/rdl_schema_0.1.json

```diff
@@ -949,6 +918,61 @@
 				"TCY"
 			]
 		},
+		"Resource": {
+			"title": "Resource",
+			"type": "object",
+			"description": "Information about an individual version of the dataset.",
+			"required": [
+				"id",
+				"title",
+				"description",
```

Comment:
Should `description` be required given that it's own description begins "Additional..."?

Reply:
yes I think it should, we should just alter the description to not begin "additional..."

Reply:
Sounds good